### DISCUSSION TITLE
Button call back

### DISCUSSION THREAD

**Discussion by flexiti** [HASwitchPlate/openHASP#707] [Q&A] [answered]
Can we use callback  from button  in custom code or only pulling metod to check state?

**Comment by fvanroie** [comment]
You can [add your own callbacks](https://docs.lvgl.io/7.11/overview/event.html#events) to an object.

**Comment by flexiti** [comment, marked as answer]
Hi,
`identifier "lv_obj_add_event_cb" is undefined`
I have the error as above - what could be the reason?

But..
`lv_obj_set_event_cb(MyOpenSw, my_event_cb); `
It seems to be working

**Reply by fvanroie** [reply]
Yeah sorry, you need to use the v7.11 API reference...
